src/components/messageCanvas/messageCanvas.tsx

```diff
@@ -36,42 +37,36 @@ function MessageCanvasElement(
   props: MessageCanvasProps,
   ref: React.Ref<HTMLDivElement>
 ) {
+  const theme = useTheme()
+
   return (
-    <Box
+    <Stack
       id={props.message.id}
       className="rustic-message-canvas"
       data-cy="message-canvas"
       ref={ref}
     >
-      <Box className="rustic-sender-info">
+      <Stack className="rustic-sender-info">
```

Comment:
You could remove the classname if you use `Stack`
```suggestion
      <Stack direction="row" alignItems="center" spacing={1}>
```